contracts/interfaces/IOrderMixin.sol

```diff
@@ -114,12 +114,13 @@ interface IOrderMixin {
     function simulate(address target, bytes calldata data) external;
 
     /**
-     * @notice Fills order's quote, fully or partially (whichever is possible)
+     * @notice Fills order's quote, fully or partially (whichever is possible).
      * @param order Order quote to fill
      * @param r R component of signature
      * @param vs VS component of signature
      * @param amount Taker amount to fill
-     * @param takerTraits Specifies threshold as maximum allowed takingAmount when takingAmount is zero, otherwise specifies minimum allowed makingAmount. Top-most bit specifies whether taker wants to skip maker's permit.
+     * @param takerTraits Specifies threshold as maximum allowed takingAmount when takingAmount is zero, otherwise specifies
+     * minimum allowed makingAmount. The 2nd (0 based index) highest bit specifies whether taker wants to skip maker's permit.
```

Comment:
It seems that docgen may generate markdown text containing "\n" and "*". We need to verify and ensure that this is not the case.

Reply:
I ran `yarn docify` and everything seems fine to me: 
<img width="919" alt="image" src="https://github.com/1inch/limit-order-protocol/assets/94478708/c704131d-c586-4a71-bd2b-b562e1cff691">